### PR TITLE
Loading indicator

### PR DESCRIPTION
**Pull request by pixelandpage**
Adds a loading indicator to the right-hand rail and to the bottom slice while the recommendations load on article pages.

### CODE REVIEW

File: client/components/affinity/main.js [hunk 1]
```diff
@@ -54,6 +56,7 @@ const buildBottom = (articles) => {
 export default (flags) => {
 	const articleId = document.querySelector('[data-content-id]').getAttribute('data-content-id');
 	if (flags.affinity) {
+		loadingIndicator.init();
```
Comment (debugwand) [inline client/components/affinity/main.js]: you also need a function to remove it. that would occur in the error handler function

File: client/components/affinity/loading-indicator.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+			const rhrLoadingPod = document.createElement('div');
+			const rhrLoadingEl = document.querySelector('.loading-indicator-rhr');
+
+			rhrLoading.className = 'loading-indicator';
+			rhrLoading.setAttribute('data-loading-indicator', 'affinity');
+			rhrLoadingInner.className = 'loading-indicator__inner';
+			rhrLoadingPod.className = 'loading-indicator__text';
+
+			rhrLoadingInner.appendChild(rhrLoadingPod);
+			rhrLoading.innerHTML += rhrLoadingInner.outerHTML;
```
Comment (debugwand) [inline client/components/affinity/loading-indicator.js]: you could have a function that creates a 'line'

File: client/components/affinity/loading-indicator.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+			const bottomLoading = document.createElement('div');
+			const bottomLoadingInner = document.createElement('div');
+			const bottomLoadingAvatar = document.createElement('div');
+			const bottomLoadingPod = document.createElement('div');
+			const bottomLoadingEl = document.querySelector('.loading-indicator-bottom');
+
+			bottomLoading.className = 'loading-indicator';
+			bottomLoading.setAttribute('data-loading-indicator', 'affinity');
+			bottomLoadingInner.className = 'loading-indicator__inner';
+			bottomLoadingAvatar.className = 'loading-indicator__avatar';
```
Comment (debugwand) [inline client/components/affinity/loading-indicator.js]: avatars are the images for a person, is this a placeholder for the story image?

File: client/components/affinity/loading-indicator.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+			bottomLoadingPod.className = 'loading-indicator__text';
+
+			bottomLoadingInner.appendChild(bottomLoadingAvatar);
+			bottomLoadingInner.appendChild(bottomLoadingPod);
+			bottomLoading.innerHTML += bottomLoadingInner.outerHTML;
```
Comment (debugwand) [inline client/components/affinity/loading-indicator.js]: could also use a function that returns a line

File: client/components/affinity/loading-indicator.scss [hunk 5]
```diff
@@ -0,0 +1,76 @@
+.loading-indicator {
+	font-family: BentonSans, sans-serif;
```
Comment (i-like-robots) [inline client/components/affinity/loading-indicator.scss]: This font should be long dead, we should only use Metric, Financier or Georgia AFAIK

**Pull request merged**